internal/cmd/commands/dev/dev.go

```diff
@@ -163,6 +164,13 @@ func (c *Command) Flags() *base.FlagSets {
 		Usage:  "Address to bind to for controller \"cluster\" purpose. If this begins with a forward slash, it will be assumed to be a Unix domain socket path.",
 	})
 
+	f.StringVar(&base.StringVar{
+		Name:   "controller-public-cluster-address",
```

Comment:
I know it makes it long, but do we want this to be `dev-controller-public-cluster-address` to mirror it with the env var?

Reply:
No, none of these flags have dev in front because they're only valid for the dev command. The reason the env vars have DEV in front is because they are otherwise not indicated which command they are related to. Similar to e.g. BOUNDARY_CONNECT_XYZ.